src/main/java/nl/tudelft/ewi/devhub/server/util/MarkDownParser.java

```diff
@@ -0,0 +1,43 @@
+package nl.tudelft.ewi.devhub.server.util;
+
+import com.google.common.html.HtmlEscapers;
+import lombok.NonNull;
+import lombok.Synchronized;
+import org.parboiled.errors.ParserRuntimeException;
+import org.pegdown.PegDownProcessor;
+
+import javax.inject.Inject;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by Douwe Koopmans on 1-6-16.
+ */
+// TODO: 2-6-16 add preview panel when writing markdown
```

Comment:
Please don't leave TODO's in the code base. These should be added as issue on the board.